src/wp-includes/default-filters.php

```diff
@@ -135,6 +135,7 @@
 add_filter( 'the_content', 'wpautop'            );
 add_filter( 'the_content', 'shortcode_unautop'  );
 add_filter( 'the_content', 'prepend_attachment' );
+add_filter( 'the_content', 'wp_resp_img'        );
```

Comment:
It looks like new filters without wp_ are fine, so let's see if we can find a name for these that are more descriptive of what you're filtering.

add_responsive_images?  responsive_image_markup?  add_responsive_images?  fill_resp_images?  (some other thing?)

Naming is hard.